src/model/formats/garmin-pilot-writer.ts

```diff
@@ -69,8 +57,9 @@ export class GarminPilotWriter {
           binders: [
             {
               uuid: uuidV4(),
```

Comment:
This one is definitively required.

Actually, one could argue that one might wish to preserve it, but I made a different decision back then for ForeFlight for safety reasons, and followed the same route for GP.

You never know how they deal with synchronization and what exactly they would do when importing a file that is meant to replace an existing binder.

Just FYI.